gramps/gui/plug/report/_styleeditor.py

```diff
@@ -395,10 +395,10 @@ def draw_graphics(self):
         self.top.get_object("line_style").set_active(g.get_line_style())
         self.top.get_object("line_width").set_value(g.get_line_width())
 
-        self.line_color = rgb2color(g.get_color())
-        self.top.get_object("line_color").set_color(self.line_color)
-        self.fill_color = rgb2color(g.get_fill_color())
-        self.top.get_object("fill_color").set_color(self.fill_color)
```

Comment:
Note that I couldn't find any use of `self.line_color`/`self.fill_color` in this file (e.g., the corollary to this function, `save_graphics`, does not use them), so I think that assignment was a mistake.